src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagStoreEntry.cs

```diff
@@ -35,6 +36,11 @@ public CustomTagStoreEntry(int key, string path, string vr, CustomTagLevel level
         /// </summary>
         public string VR { get; set; }
 
+        /// <summary>
+        /// Private Creator of this tag. Only apply to private tag.
```

Comment:
>/// Private Creator of this tag. Only apply to private tag. [](start = 8, length = 59)

Can you explain what a private creator is and what the use of this field is?

Reply:
Hi Sneha, this is the "Private Creator" I'm talking about in today's syncup.
Private Tag is widely used in industry, it likely to happen that 2 users are using dicom tag of same groupId and elementId, but means different. The private creator is description of implementer, could help distinguish that. 
More details can be find in standard:  http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_7.8.html

Reply:
Sorry if that was unclear - i meant add a description in the summary!

---
In reply to: [595574340](https://github.com/microsoft/dicom-server/pull/644#discussion_r595574340) [](ancestors = 595574340)

Reply:
I will add these information to summary.